### DISCUSSION TITLE
Stream Playlist VOD on Twitch

### DISCUSSION THREAD

**Discussion by romanhopeok** [ant-media/Ant-Media-Server#4977] [Q&A]
Hi how are you?

I've created a playlist stream on Twitch.

I have 4 videos. The thing is that every time one of the video finishes when the next one starts it also start a new stream on twitch.

It's not a big continuous stream. Videos are 1 hour length so it's also very annoying every hour followers receive a notification about the stream has started.

Is there a way to fix it?

Kind regards,
Roman

**Comment by lastpeony** [comment]
Yes, there is a way to fix this issue. You need to make sure that the videos are played as a single continuous stream, rather than starting a new stream every time a video ends. This can be achieved by using a playlist feature.

 you can create a playlist of the videos you want to stream and configure the server to play the playlist continuously without starting a new stream every time a video ends.

Here are the steps you can follow to create a playlist in Ant Media Server:

Log in to your Ant Media Server dashboard.
Click on the "Live App" tab.
Click on the "New Live Stream" button.
Click on the "Playlist" tab.
Give your playlist a name
Type paths to your video ex(/home/videos/test.mp4)
Repeat step 6 for each video you want to add to the playlist.
Click on the "Create" button to create playlist livestream.
![image](https://user-images.githubusercontent.com/24750845/230299418-78d22080-b67e-4530-9c9d-bfaae761214d.png)

Once your playlist live stream is created, you can configure AMS to use the playlist as the source for your stream. This will ensure that the videos are played as a single continuous stream, without starting a new stream every time a video ends.

By following these steps, you should be able to fix the issue you are experiencing with your Twitch playlist stream.

**Reply by romanhopeok** [reply]
That's exactly what I did. So I don't know what the problem could it be.

**Reply by mekya** [reply]
hi @lastpeony and @romanhopeok 
Thank you for the discussion. 

I think this is a known issue. In the playlist, it just stops and starts a new stream when it switches to another item in the list. So we have this issue https://github.com/ant-media/Ant-Media-Server/issues/4468

We'll schedule it when we have some availability for this issue.

**Reply by mekya** [reply]
Hi @lastpeony and @romanhopeok ,
I just want to update you. We've provided a workaround solution for this problem. 

Check this out -> https://github.com/orgs/ant-media/discussions/6158
